releases/release-1.24/README.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "Kubernetes 1.24 Release Information"
+weight: 98
+slug: "release"
+aliases: [ "/release" ]
+description: |
+  Information regarding the current release cycle including important dates,
+  Release Team contact information, tracking spreadsheets and more!
+---
+
+# Kubernetes 1.24
+
+#### Links
+
+* [This document](https://git.k8s.io/sig-release/releases/release-1.24/README.md)
+* [Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.24/release-team.md)
+* [Meeting Minutes](http://bit.ly/k8s124-releasemtg) (join [release-team@] to receive meeting invites)
+* [v1.24 Release Calendar][k8s124-calendar]
+* Contact: [#sig-release] on slack, [release-team@] on e-mail
+* [Internal Contact Info] (accessible only to members of [release-team@])
+
+#### Tracking docs
+
+* [Enhancements Tracking Sheet](https://bit.ly/k8s124-enhancements)
+* [Feature blog Tracking Sheet](http://bit.ly/k8s124-feature-blog)
+* [Bug Triage Tracking Sheet](https://bit.ly/124-bug-triage-tracking)
+* [CI Signal Project Board](https://github.com/orgs/kubernetes/projects/11)
+* [CI Signal Tracking Log](https://docs.google.com/spreadsheets/d/1ns94ZAH4PIvzhpwU1SGPyl9ek_p4-_Be/)
+* [Retrospective Document][Retrospective Document]
+* [kubernetes/sig-release v1.24 milestone](https://github.com/kubernetes/kubernetes/milestone/56)
+
+#### Guides
+
+* [Targeting Issues and PRs to This Milestone](https://git.k8s.io/community/contributors/devel/sig-release/release.md)
+* [Triaging and Escalating Test Failures](https://git.k8s.io/community/contributors/devel/sig-testing/testing.md#troubleshooting-a-failure)
+
+## TL;DR
+
+The 1.24 release cycle is proposed as follows:
+
+- **Monday 3rd January 2022**: Week 1 - Release cycle begins
```

Comment:
I know the [KEP proposes the same](https://github.com/kubernetes/enhancements/tree/master/keps/sig-release/2572-release-cadence#proposal), but how about starting the cycle on January 10th?

Reply:
Absolutely agree.